components/events/EventCard.vue

```diff
@@ -1,8 +1,8 @@
 <template>
-  <a
-    :href="to"
+  <component
```

Comment:
An [`<a>` tag without the `href` is semantically correct](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element) and means a placeholder for when a may be expected, as is the case with event cards:
> If the a element has no href attribute, then the element represents a placeholder for where a link might otherwise have been placed, if it had been relevant, consisting of just the element's contents.

 We just need to realize that if the `to` property is not provided we should go for the link tag without including any other link-related property (even `href`).